aws-android-sdk-iot/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -779,20 +805,43 @@ public void onMessageArrived(String topic, byte[] data) {
 
     @Test
     public void mqttWebSocketReconnect() throws Exception {
+        websocketReconnectionTest(IAM);
+    }
 
+    @Test
+    public void testCustomAuthReconnect() throws Exception {
+        websocketReconnectionTest(CUSTOM_AUTH);
```

Comment:
`testWebsocketReconnectWithIam`, `testWebsocketReconnectWithCustomAuth`

Something like that